### PR TITLE
feat(secondary-nav): create secondary-nav and its stories

### PR DESCRIPTION
**Pull request by nathanbowang**
Create SecondaryNav component.
Create the component's stories.
Install Storybook a11y add-on.
Update Storybook configuration.

### CODE REVIEW

File: components/Header/SecondaryNav/index.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+export default function SecondaryNav(): JSX.Element {
+  return (
+    <nav aria-label="Secondary">
+      <ul className="capitalize">
+        <li>
+          <a href="#section-one" target="_self">
```
Comment (chasm) [inline components/Header/SecondaryNav/index.tsx:6]: Do we need `target` if it's set to `_self`?

**Review by chasm — APPROVED** [review]

File: components/Header/SecondaryNav/index.tsx (hunk 1 above)
Comment (nathanbowang) [reply components/Header/SecondaryNav/index.tsx:6]: The reason is Storybook override target value in head:
```html
<head>
  <base target="_parent" />
</head>
```
When user clicks on a link in Storybook app webpage, a new tab would pop up. I have tried [add a new base element](https://storybook.js.org/docs/react/configure/story-rendering#adding-to-head) to Storybook's head and got this:
```html
<head>
  <base target="_parent" />
  <base target="_self" />
</head>
```
But no luck, still pup up a new tab. However, add `target='_self'` is also annoying. What do you think?

Comment (chasm) [reply components/Header/SecondaryNav/index.tsx:6]: I was just wondering. But good point -- this is only for Storybook. So it's fine.

When using `target` with anything other than self, it is best for security reasons to include `rel="noopener noreferrer"`:

https://web.dev/external-anchors-use-rel-noopener/

Comment (chasm) [reply components/Header/SecondaryNav/index.tsx:6]: Worth looking at: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel

File: .storybook/preview.js [hunk 2]
```diff
@@ -8,4 +8,5 @@ export const parameters = {
       date: /Date$/,
     },
   },
+  layout: 'fullscreen',
```
Comment (reubenberghan) [inline .storybook/preview.js]: It is probably more useful that this is done at the story level than across the board. As though a nav might be better with the `fullscreen` layout we get a better picture of most other components with space around them.

File: components/Header/SecondaryNav/index.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import SecondaryNav from '.'
+import type { Meta, Story } from '@storybook/react'
+import { ReactElement } from 'react'
```
Comment (reubenberghan) [inline components/Header/SecondaryNav/index.stories.tsx]: Missing `type` here 😁

**Review by reubenberghan — CHANGES_REQUESTED** [review]
Nice work. A couple of small things though 😁

**Review by reubenberghan — APPROVED** [review]

**Pull request merged**